packages/web_ui/src/components/InstancesPage.jsx

```diff
@@ -18,7 +18,9 @@ export default function InstancesPage() {
 		return result["list"].map(item => ({
 			key: item["id"],
 			"Name": item["name"],
-			"Assigned Slave": item["assigned_slave"],
+			"Public Address": item["public_address"] + ":" + item["game_port"],
+			"Assigned Slave": item["assigned_slave_name"],
+			"Assigned Slave ID": item["assigned_slave"],
```

Comment:
We probably don't need to show assigned_slave when we have assigned_slave_name

Reply:
yep, but I think `assigned_slave` should still be exposed to facilitate troubleshooting in some cases

Reply:
It's already exposed in the instance view so I see no reason to expose it here. The web interface doesn't generally show IDs because they are long random numbers.

Reply:
so do you think that only the `slave_name` should be displayed in the web interface instead of the `slave_id` again?

Reply:
Yes, long random numbers don't belong in the web interface. It could be shown in a details page for each slave, but that's a later problem.

Reply:
ok, i deleted `assigned_slave`  from WebUI column and push commit